src/main/scala/fi/oph/koski/raportit/PerusopetuksenLisäopetusOppijamäärätRaportti.scala

```diff
@@ -0,0 +1,109 @@
+      koulukoti = r.<<
+    )
+  )
+
+  def build(oppilaitosOids: Set[String], date: Date)(implicit u: KoskiSession): DataSheet = {
+    val raporttiQuery = query(oppilaitosOids, date).as[PerusopetuksenLisäopetusOppijamäärätRaporttiRow]
+    val rows = runDbSync(raporttiQuery, timeout = 5.minutes)
+    DataSheet(
+      title = "Suoritukset",
+      rows = rows,
+      columnSettings = columnSettings
+    )
+  }
+
+  private def query(oppilaitosOids: Set[String], date: Date)(implicit u: KoskiSession) = {
+    sql"""
+    select
+      oo.oppilaitos_nimi,
+      opetuskieli_koodisto.nimi as opetuskieli,
+      count(distinct oo.opiskeluoikeus_oid) as oppilaita,
+      count(distinct (case when r_henkilo.aidinkieli not in ('fi', 'sv', 'se', 'ri', 'vk') then oo.opiskeluoikeus_oid end)) as vieraskielisiä,
+      count(distinct (case when                   vaikeasti_vammainen and pidennetty_oppivelvollisuus then oo.opiskeluoikeus_oid end)) as pidennettyOppivelvollisuusJaVaikeastiVammainen,
+      count(distinct (case when vammainen and not vaikeasti_vammainen and pidennetty_oppivelvollisuus then oo.opiskeluoikeus_oid end)) as pidennettyOppivelvollisuusJaMuuKuinVaikeimminVammainen,
+      count(distinct (case when                   vaikeasti_vammainen and not (pidennetty_oppivelvollisuus and erityinen_tuki) then oo.opiskeluoikeus_oid end)) as virheellisestiSiirretytVaikeastiVammaiset,
+      count(distinct (case when vammainen and not vaikeasti_vammainen and not (pidennetty_oppivelvollisuus and erityinen_tuki) then oo.opiskeluoikeus_oid end)) as virheellisestiSiirretytMuutKuinVaikeimminVammaiset,
+      count(distinct (case when erityinen_tuki then oo.opiskeluoikeus_oid end)) as erityiselläTuella,
+      count(distinct (case when majoitusetu then oo.opiskeluoikeus_oid end)) as majoitusetu,
+      count(distinct (case when kuljetusetu then oo.opiskeluoikeus_oid end)) as kuljetusetu,
+      count(distinct (case when sisaoppilaitosmainen_majoitus then oo.opiskeluoikeus_oid end)) as sisäoppilaitosmainenMajoitus,
+      count(distinct (case when koulukoti then oo.opiskeluoikeus_oid end)) as koulukoti
+    from r_opiskeluoikeus oo
+    join r_henkilo on r_henkilo.oppija_oid = oo.oppija_oid
+    join r_paatason_suoritus on r_paatason_suoritus.opiskeluoikeus_oid = oo.opiskeluoikeus_oid
+    join r_opiskeluoikeus_aikajakso aikajakso on aikajakso.opiskeluoikeus_oid = oo.opiskeluoikeus_oid
+    left join r_organisaatio_kieli on r_organisaatio_kieli.organisaatio_oid = oo.oppilaitos_oid
+    left join r_koodisto_koodi opetuskieli_koodisto
+      on opetuskieli_koodisto.koodisto_uri = split_part(split_part(r_organisaatio_kieli.kielikoodi, '#', 1), '_', 1)
+      and opetuskieli_koodisto.koodiarvo = split_part(r_organisaatio_kieli.kielikoodi, '#', 2)
+    where oo.oppilaitos_oid in (#${toSqlListUnsafe(oppilaitosOids)})
+      and oo.koulutusmuoto = 'perusopetuksenlisaopetus'
+      and r_paatason_suoritus.vahvistus_paiva is null
```

Comment:
varmistan vain et jos opiskelija on läsnäoleva ja sen suorituksella on vahvistus niin sitä ei ole tarkoitus huomioida raportille?

Reply:
Näin tehtiin perusopetuksen raportilla, mut varmistanpa vielä Juholta tän osalta